gemet/thesaurus/import_spreadsheet.py

```diff
@@ -0,0 +1,304 @@
+from openpyxl import load_workbook
+from openpyxl.utils.exceptions import InvalidFileException
+
+from django.db import transaction
+from django.utils import timezone
+
+from gemet.thesaurus import PENDING, PUBLISHED
+from gemet.thesaurus.models import (
+    Namespace, Version, Concept, Property, PropertyType, Relation, Language
+)
+from gemet.thesaurus.utils import get_new_code, get_search_text
+
+
+class ImportError(Exception):
+    pass
+
+
+def namespace_for(property_type_name):
+    heading_for = {
+        'broader': 'Concepts',
+        'group': 'Groups',
+    }
+    return Namespace.objects.get(
+        heading=heading_for[property_type_name]
+    )
+
+
+def row_dicts(sheet):
+    rows = sheet.rows
+
+    mandatory_columns = {
+        "Term", "Definition", "Definition reference"
+    }
+    optional_columns = {
+        "Alt Label", "Abbreviation/Alt Label", "Synonym/Alt Label",
+        "Broader concept", "Broader URI", "Group", "Note"
+    }
+    optional_columns.add(sheet.title)
```

Comment:
Because in the sheets with translations, the translated value is in the column whose name is the code of the language. For example, in the FR sheet, there is a column named "FR", with the French name of the concept.

Reply:
This sounds unnecessary, you could simply use prefLabel for all translations? Or "Target   language", as it was in the original file provided on the ticket.

Reply:
The original file provided in the ticket uses "BG" for Bulgarian, "CS" for Czech, that's why I used this convention. But indeed most other sheets use "Target language" instead. I will change the format so all sheets have "Target language".